Run job by id (#65)
PrefectHQ/prefect-databricks: README.md

> # prefect-databricks
> 
> ## Welcome!
> 
> Prefect integrations interacting with Databricks
> 
> The tasks within this collection were created by a code generator using the service's OpenAPI spec.
> 
> The service's REST API documentation can be found [here](https://docs.databricks.com/dev-tools/api/latest/index.html).

Reviewer: We can remove this section. The information regarding generated tasks is in the __Resources__ section.
```suggestion
```

    auto_scale={"min_workers": 1, "max_workers": 2}
    ```

> Another example in which you do have an existing Databricks job and wish to trigger it by inputting the job id is below.

Reviewer: ```suggestion
If you have an existing Databricks job, you can run it using `jobs_runs_submit_by_id_and_wait_for_completion`:
```
